src/nodes.coffee

```diff
@@ -1803,30 +1809,19 @@ exports.JSXElement = class JSXElement extends Base
       if content instanceof StringLiteral
         [new JSXText content]
       else # StringWithInterpolations
-        for element in @content.unwrapAll().extractElements o, includeInterpolationWrappers: yes
+        for element in @content.unwrapAll().extractElements o, includeInterpolationWrappers: yes, isJsx: yes
```

Comment:
The changes here and in `StringWithInterpolations` below are related to how we handle AST generation for empty interpolations (either in strings eg `"#{}"` or in JSX expressions eg `<div>{}</div>`)

The main change is that for empty interpolations in strings, instead of generating a `null` AST (in the `expressions` list of the `TemplateLiteral` AST node), we generate a new `EmptyInterpolation` AST node type

`EmptyInterpolation` doesn't exist in Babel-style AST because JS doesn't allow empty interpolations eg `` `${}` `` is a syntax error

So it's Coffeescript-specific. But it mimics `JSXEmptyExpression`, which does exist in Babel-style ASTs (since empty JSX expressions are allowed)

The main reason it's better to have an actual AST node representing the empty interpolation (other than just for consistency with `JSXEmptyExpression`) is that it makes handling comments inside the empty interpolations (eg `"#{###here###}"`) much more straightforward (added AST tests for empty interpolations with/without comments for both strings and JSX expressions)